sitzverteilung-frontend/src/components/basedata/BaseDataForm.vue

```diff
@@ -35,7 +35,7 @@
       <v-col>
         <v-number-input
           v-model="baseData.targetSize"
-          :rules="[FieldValidationRules.Required]"
+          :rules="[...getTargetSizeRules]"
```

Comment:
Ich würde es tatsächlich eher inline machen, da das Feld ja sonst keine weiteren Regeln hat.


```suggestion
          :rules="areFieldsRequired ? [FieldValidationRules.Required] : []"
```